Formula/c/codequery.rb

```diff
@@ -22,19 +22,16 @@ class Codequery < Formula
   fails_with gcc: "5"
 
   def install
-    args = std_cmake_args
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```

Comment:
Since we don't have `-G Ninja` here, is Ninja really used? If not can we drop the `ninja` dependency?